pkg/cmd/serve.go

```diff
@@ -158,7 +158,6 @@ func RegisterServeFlags(cmd *cobra.Command, config *server.Config) error {
 
 	experimentalFlags := nfs.FlagSet(BoldBlue("Experimental"))
 	// Flags for experimental features
-	experimentalFlags.BoolVar(&config.EnableExperimentalLookupResources, "enable-experimental-lookup-resources", true, "enables the experimental version of the lookup resources API")
```

Comment:
don't remove, as it will break callers. Instead, deprecate it and if someone gives a value of `false`, issue a warning that the flag will be removed in the future